preprocessing/tabular_data/preprocessing_tabular_data.ipynb

```diff
@@ -0,0 +1,268 @@
+{
```

Comment:
<p>Remove "For simplicity purposes"</p><p> </p><p>I think the first sentence should say "we use storemagic to persist all relevant variables so <em>they</em> can" (not "it")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/amazon-sagemaker-examples/pull/1466/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>